pipeline/inputs/opentelemetry.md

```diff
@@ -50,13 +50,13 @@ __OTLP/GRPC__
 
 ## Getting started
 
-The OpenTelemetry plugin currently supports the following telemetry data types:
+The OpenTelemetry input plugin currently supports the following telemetry data types:
 
-|     Type    |    HTTP/JSON  |   HTTP/Protobuf |
-| ----------- | ------------- | --------------- |
-|    Logs     |     Stable    |      Stable     |
-|    Metrics  | Unimplemented |      Stable     |
-|    Traces   | Unimplemented |      Stable     |
+|     Type    |   HTTP1/JSON  |  HTTP1/Protobuf | HTTP2/GRPC |
+| ----------- | ------------- | --------------- | ---------- |
+|    Logs     |     Stable    |      Stable     |   Stable   |
+|    Metrics  | Unimplemented |      Stable     |   Stable   |
+|    Traces   | Unimplemented |      Stable     |   Stable   |
```

Comment:
Cutting down on additional spaces to help with line wraps and to be consistent with table formatting standards.

```suggestion
| Type    | HTTP1/JSON | HTTP1/Protobuf | HTTP2/GRPC |
| ------- | ---------- | -------------- | ---------- |
| Logs    | Stable | Stable | Stable |
| Metrics | Unimplemented | Stable | Stable |
| Traces  | Unimplemented | Stable | Stable |
```